packit/api.py

```diff
@@ -275,8 +276,11 @@ def _prepare_files_to_sync(
     ) -> List[RawSyncFilesItem]:
         if self.package_config.sync_changelog:
             return raw_sync_files
-
-        comment = f"- new upstream release: {full_version}"
+        comment = (
+            self.up.local_project.git_project.get_release(name=full_version).body
+            if self.package_config.copy_upstream_release_description
+            else f"- new upstream release: {full_version}"
```

Comment:
can we actually "else" to the same behaviour we do for SRPMs - get a list of commits from the last tag?

Reply:
yes, I was thinking about this, but that would change the behaviour for all users..but that's probably not a problem, so I will implement it

Reply:
yeah, that's a good point, I recall we had an issue for that so was thinking we could do it here